### ISSUE TITLE
OWSpectra: customize plot style and add annotations

### ISSUE THREAD

**Issue by pisarik** [Quasars/orange-spectroscopy#636]
Hi!

In Scatter Plot you can easily change font size of axis labels, ticks, title and so on... But in Spectra widget you don't have that button to open a window with style settings. Is it possible to add it? I believe it should be simple.

We also have few suggestions that would be very handy for a figure preparation for publication, but they might require more work: 
1) Add peak annotation (with possibility to add some text) 
![image](https://user-images.githubusercontent.com/3684845/196232504-5939aa88-aea1-403a-95b1-f665c43d5290.png)
2) Specify at least a ticks step (e.g. have a tick every 5 $cm^{-1}$)

What do you think about it?

**Comment by markotoplak** [comment]
Hi, nice to hear from you. :)

Peak annotation is already possible. Also, regarding plot parameters, try #605 and comment there. :)

**Comment by pisarik** [comment]
It's been done for like six months already, I have to update the package more often!:) Thanks! This is exactly what we've been looking for. 

Many people from our group would love to produce images right in the Orange, so I think I can try to gather some opinions on what would type of visual parameters would be nice to have. Do you think #605 is a good place to list some requests?

**Comment by markotoplak** [comment]
Yes, feel free to suggest anything... Then we'll decide regarding what is usable and easily implementable.
